### PR TITLE
doctest for overloading show for Day(1) printing as "1 day"

### PR DESCRIPTION
**Pull request by ggggggggg**
I always forget exactly how to overload printing of types, and I find an example much more helpful than a lot of prose. Plus I suspect `Day` used to be defined in base, but no longer is. This makes the existing text make more sense.

### CODE REVIEW

**Comment by ggggggggg** [comment]
Is there something subtly wrong with my formatting? Not finding the error messages helpful.

File: base/multimedia.jl [hunk 1]
```diff
@@ -104,6 +104,18 @@ for that case. If a type benefits from custom human-readable output though,
 `show(::IO, ::MIME"text/plain", ::T)` should be defined. For example, the `Day` type uses
 `1 day` as the output for the `text/plain` MIME type, and `Day(1)` as the output of 2-argument `show`.
 
+# Examples
+```jldoctest
+julia> struct Day
+       n::Int
+       end
+
+julia> Base.show(io::IO, ::MIME"text/plain", d::Day) = print(io, "$(d.n) day")
```
Comment (jmert) [inline base/multimedia.jl:113]:
```suggestion
julia> Base.show(io::IO, ::MIME"text/plain", d::Day) = print(io, "\$(d.n) day")
```
I expect this is the fix you're looking for.

Comment (jmert) [inline base/multimedia.jl:113]:
```suggestion
julia> Base.show(io::IO, ::MIME"text/plain", d::Day) = print(io, "\$(d.n) day")
```
I expect this is the fix you're looking for.

Comment (mcabbott) [reply base/multimedia.jl:113]: No need to interpolate:
```suggestion
julia> Base.show(io::IO, ::MIME"text/plain", d::Day) = print(io, d.n, " day")
```

**Pull request merged**